app/views/chatrooms/index.html.erb

```diff
@@ -1,6 +1,4 @@
-<div class="row col-md-8 col-md-offset-2" id="chatroom-title">
-  <h1>Chatrooms</h1>
-</div>
+<h1>Chatrooms</h1>
```

Comment:
I think we actually want this inside the div to make sure it's width is properly constrained.